### PR TITLE
Update to ratpack 0.9.18

### PR DESCRIPTION
**Pull request by charliek**
Still a work in progress since the static files are not currently working.

### CODE REVIEW

File: hopscotch-web/src/main/java/charliek/hopscotch/web/HopscotchApp.java [hunk 1]
```diff
@@ -28,17 +33,20 @@ public static void main(String[] args) throws Exception {
 			.registry(Guice.registry(b -> b
 				.bindInstance(ConfigData.class, configData)
 				.module(HopscotchModule.class)
```
Comment (charliek) [inline hopscotch-web/src/main/java/charliek/hopscotch/web/HopscotchApp.java:35]: If the hopscotch module is commented out the static files work as expected.  Or if the ApiHandlerDecorator is commented out things also work as expected.

File: hopscotch-web/src/main/java/charliek/hopscotch/web/ApiHandlerDecorator.java [hunk 2]
```diff
@@ -8,8 +8,8 @@
 public class ApiHandlerDecorator implements HandlerDecorator {
 	@Override
 	public Handler decorate(Registry serverRegistry, Handler rest) throws Exception {
-		return Handlers.chain(serverRegistry, c -> {
+		return Handlers.chain(Handlers.chain(serverRegistry, c -> {
 			c.get("api/timed", TimedHandler.class);
-		});
+		}), rest);
```
Comment (johnrengelman) [inline hopscotch-web/src/main/java/charliek/hopscotch/web/ApiHandlerDecorator.java:13]: Good, you found this. Was just about to reply to your forum post.

File: hopscotch-web/src/main/java/charliek/hopscotch/web/HopscotchApp.java [hunk 3]
```diff
@@ -28,17 +33,20 @@ public static void main(String[] args) throws Exception {
...
 					public void onStart(StartEvent event) throws Exception {
 						RxRatpack.initialize();
 					}
 				})
-				.bind(ErrorHandler.class, DefaultDevelopmentErrorHandler.class)))
+				.bind(ClientErrorHandler.class, DefaultDevelopmentErrorHandler.class)))
 			.handlers(c -> {
 				c.prefix("static", nested ->
-					nested.assets("/static", "index.html"));
+					nested.files(f -> f.dir("static").indexFiles("index.html")));
```
Comment (johnrengelman) [inline hopscotch-web/src/main/java/charliek/hopscotch/web/HopscotchApp.java:49]: You can actually simply this starting from `prefix` to be:

```
c.files(f -> f.path("static").dir("static").indexFiles("index.html"))
```

File: hopscotch-web/src/main/java/charliek/hopscotch/web/ApiHandlerDecorator.java (hunk 2 above)
Comment (charliek) [inline hopscotch-web/src/main/java/charliek/hopscotch/web/ApiHandlerDecorator.java:13]: Thanks for taking a look. I debated on deleting the forum post, but I figured it was just noise and was hoping that maybe nobody spent time on it yet.  I'm slowly getting the hang of it.

**Pull request merged**